plugins/inputs/opcua/README.md

```diff
@@ -155,6 +155,15 @@ This example group configuration has two groups with two nodes each:
   ]
 ```
 
+## Connection Service
+
+This plugin actively reads to retrieve data from the OPC server. 
+This is done every `interval`. 
+
+## Metrics
+
+The metrics collected by this input plugin will depend on the
+configured `nodes` and `group`.
 ## Metrics
 
 Metrics are produced according to the defined node ID and group configuration.
```

Comment:
Double Metrics section by accident?

Reply:
Something went wrong during the merge apparently.